infra/token-transfer-server/src/scripts/send_welcome_token.js

```diff
@@ -81,6 +72,23 @@ async function sendWelcomeEmail(email) {
   logger.info('Email sent')
 }
 
+/* Sends emails to a single user or all users depending on args.
+ */
+async function main(config) {
+  if (config.email) {
+    const user = await User.findOne({ where: { email: config.email } })
+    if (!user) {
+      logger.error('User with that email does not exist')
+      process.exit()
+    }
+    sendWelcomeEmail(user)
+  } else {
+    logger.info('Sending welcome email to all users')
+    const users = await User.findAll()
+    users.map(sendWelcomeEmail)
```

Comment:
Should this await otherwise it will fire hundreds of promises and sendgrid may rate limt ?